core/common/src/main/java/alluxio/util/io/PathUtils.java

```diff
@@ -283,17 +321,19 @@ public static String subtractPaths(String path, String prefix) throws InvalidPat
    * @throws InvalidPathException when the path or prefix is invalid
    */
   public static boolean hasPrefix(String path, String prefix) throws InvalidPathException {
```

Comment:
A short hand can be `if (prefix.equals("/") || prefix.equals("")) return true`

Reply:
I checked the condition in `validatePath` and found that if path is "", then it is an invalid path. The condition is as follow:
```golang
boolean invalid = (path == null || path.isEmpty());
...
```
So we only need to check if `prefix` equals to "/".

Reply:
Good catch, pls add to the comment that prefix="" is invalid